assets/eip-5050/Spells.sol

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.0;
+
+import {Base64} from "@openzeppelin/contracts/utils/Base64.sol";
+import {SafeMath} from "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import {ERC721} from "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import {IERC721} from "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
```

Comment:
```suggestion
```
Most of these seem very far out of scope of this EIP.  Ownable, for example, is completely unrelated to this EIP IIUC and can be left out entirely.  Similarly, SafeMath is an implementation detail best left up to the reader rather than being included here.